client/src/app.js

```diff
@@ -148,6 +148,37 @@
       console.warn('ERROR(' + err.code + '): ' + err.message);
     }
 
+
+    var sessionHeroOrder = $window.JSON.parse($window.sessionStorage.getItem('heroOrder'));
+    if( sessionHeroOrder !== null ) {
+      heroFactory.setOrder(sessionHeroOrder);
+      console.log('default hero order loaded');
+    }
+
+    var sessionReqOrder = $window.JSON.parse($window.sessionStorage.getItem('requesterOrder'));
+    if( sessionReqOrder !== null ) {
+      requesterFactory.setOrder(sessionReqOrder);
+      console.log('default req order loaded');
+    }
+
+    $rootScope.$watch(function(){
+      return heroFactory.getOrder();
+    }, function watchCallback(newval, oldval){

```

Comment:
`newval` and `oldval` should use camel case. `newVal` `oldVal`

missing space after `watchCallback(newval, oldval)`